src/docker.test.ts

```diff
@@ -72,10 +72,10 @@ describe("Docker images", (): void => {
     expect(core.getInput).nthCalledWith<[string, InputOptions]>(1, "key", {
```

Comment:
Commit message: the scope can be more specific in this case (e.g., `docker`) since only one file is modified.